MAVProxy/mavproxy.py

```diff
@@ -949,9 +954,11 @@ def set_stream_rates():
         else:
             rate = mpstate.settings.streamrate2
         if rate != -1 and mpstate.settings.streamrate != -1:
-            master.mav.request_data_stream_send(mpstate.settings.target_system, mpstate.settings.target_component,
-                                                mavutil.mavlink.MAV_DATA_STREAM_ALL,
-                                                rate, 1)
+            # Send to all detected vehicles in this link
```

Comment:
If we have a lot of vehicles on this link this structure is going to cause a massive blurt of requests to all be sent at once, which may be a problem from a max-latency perspective.

Reply:
I could put in a time.sleep(0.01)?

Reply:
Not here - pretty sure this is MAVProxy's main loop.

*If* it becomes an issue it will probably be some sort of `last_rate_set_ms(sysid, compid)` or something gradually iterating through the vehicles on the link, setting each in turn.

This isn't something that needs to be fixed on this PR - just something to watch out for.

Reply:
Otherwise, I could get fancy and only send a few ``request_data_stream_send`` each time the function's called...